packages/uniforms-material/src/DateField.tsx

```diff
@@ -3,9 +3,11 @@ import TextField, { TextFieldProps } from '@material-ui/core/TextField';
 import React from 'react';
 import { FieldProps, connectField, filterDOMProps } from 'uniforms';
 
+export type AllowedDateTypes = 'date' | 'datetime-local';
```

Comment:
I wouldn't export this type as we don't do that for any of them and inline it instead.